src/gui/BandSplitter/BandSplitterChyron.cpp

```diff
@@ -0,0 +1,95 @@
+#include "BandSplitterChyron.h"
+
+namespace gui::band_splitter
+{
+BandSplitterChyron::BandSplitterChyron (chowdsp::PluginState& pluginState,
+                                        dsp::band_splitter::Params& bandSplitterParameters,
+                                        const chowdsp::HostContextProvider& hcp)
+    : state (pluginState),
+      bandSplitterParams (bandSplitterParameters),
+      hostContextProvider (hcp)
+{
+    updateValues();
+    callbacks +=
+        {
+            pluginState.addParameterListener (*bandSplitterParams.threeBandOnOff,
+                                              chowdsp::ParameterListenerThread::MessageThread,
+                                              [this]
+                                              {
+                                                  updateValues();
+                                              }),
+            pluginState.addParameterListener (*bandSplitterParams.fourBandOnOff,
+                                              chowdsp::ParameterListenerThread::MessageThread,
+                                              [this]
+                                              {
+                                                  updateValues();
+                                              }),
+        };
+}
+
+void BandSplitterChyron::updateValues()
+{
+    const auto reset = [this]
+    {
+        cutoffSliderLow.reset();
+        cutoffSliderMid.reset();
+        cutoffSliderHigh.reset();
+    };
+
+    reset();
+
+    auto bandState = bandSplitterParams.getCurrentBandState();
+    cutoffSliderLow.emplace (state, bandSplitterParams.cutoff.get(), &hostContextProvider);
+    cutoffSliderLow->setName ("Cutoff Low");
+    addAndMakeVisible (*cutoffSliderLow);
+
+    if (bandState != dsp::band_splitter::BandState::TwoBands)
+    {
+        cutoffSliderMid.emplace (state, bandSplitterParams.cutoff2.get(), &hostContextProvider);
+        cutoffSliderMid->setName ("Cutoff Mid");
```

Comment:
I wonder if we might want to call this "Cutoff High" for 3-band mode, and then change it to "Cutoff Mid" only for the 4-band mode?